AppInspector/rules/default/cloud_services/web_analytics.json

```diff
@@ -3,9 +3,7 @@
     "name": "Miscellaneous: Analytics Service",
     "id": "AI004200",
     "description": "Miscellaneous: Analytics Service",
-    "tags": [
-      "CloudServices.Web.Analytics.Facebook"
-    ],
+    "tags":[ "CloudServices.Web.Analytics.Facebook" ],
```

Comment:
I think just `CloudServices.Web.Analytics`, since the pattern doesn't look to be FB-specific.